src/core/qfieldcloudprojectsmodel.cpp

```diff
@@ -2170,6 +2252,16 @@ void QFieldCloudProjectsModel::loadProjects( const QJsonArray &remoteProjects, b
       }
     }
 
+    if ( projectDetails.contains( "localized_datasets" ) )
+    {
+      const QVariantList localizedDatasets = projectDetails.value( "localized_datasets" ).toList();
+      for ( const QVariant &localizedDataset : localizedDatasets )
+      {
+        const QVariantMap localizedDatasetDetails = localizedDataset.toMap();
+        cloudProject->localizedDatasets << localizedDatasetDetails.value( "datasource" ).toString().mid( 10 );
```

Comment:
I do not follow this bit here? You skip the first 10 chars because they are `localized:` I guess?

I guess a comment can be helpful here.

Docs reference: https://doc.qt.io/qt-6/qstring.html#mid

Reply:
See comment below